### ISSUE TITLE
Create skeleton Rust project and hook up to travis-ci

### ISSUE THREAD

**Issue by ryandoyle** [performancecopilot/hornet#1]

**Comment by suyash** [comment]
Please see https://github.com/japaric/trust

Please add macOS builds for travis, and anything else that you think might be useful so we can close this

**Comment by saurvs** [comment]
That project achieves cross-architecture testing through QEMU, which would probably slow down the CI. And the only architecture specific aspect to this project I can think of is endianess - should we default to system endian or little endian? I think Speed defaults to little, libpcp_mvm not sure. @ryandoyle

**Comment by suyash** [comment]
@saurvs for speed I defaulted to Litte Endian (https://github.com/performancecopilot/speed/blob/master/bytewriter/bytewriter.go#L10) and decided that will create `bytewriter_os_arch.go` overriding `byteorder` with `BigEndian` if it was ever needed. I think we can take a similar approach for now. I just linked the project in case I might have missed something. In general, I don't think we need to test for specific architectures for now, and testing on macOS should be enough.
